src/App.tsx

```diff
@@ -5,7 +5,24 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (movie: Movie) => {
+    let isMovieInList = false;
+
+    movies.forEach(item => {
```

Comment:
it is the perfect case to use `some` method instead of `forEach` (and assign result to const variable)